Sources/Command/CommandRunner.swift

```diff
@@ -156,6 +156,9 @@ public struct CommandRunner: CommandRunning, Sendable {
                         if data.count > 0 {
                             stdoutQueue.async {
                                 continuation.yield(.standardOutput([UInt8](data)))
+                                if let output = String(data: data, encoding: .utf8) {
+                                    logger?.error("\(output)")
+                                }
```

Comment:
```suggestion
                                if let output = String(data: data, encoding: .utf8) {
                                    logger?.info("\(output)")
                                }
```

You probably meant to use `logger?.info` instead of `error`. But as mentioned in the review comment, I'd probably consider relegating this to `logger?.debug`.

In case we do relegate this `logger?.debug`, we can add more information associated with the log to make it easier to couple the `output` with the command that was run, such as by doing this:
```suggestion
                                if let output = String(data: data, encoding: .utf8) {
                                    logger?.debug("Command \(arguments.joined()) succeeded with the following output: \(output)")
                                }
```